modules/cloudtrail/main.tf

```diff
@@ -0,0 +1,332 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  trail_name = "ksoc-discovery"
```

Comment:
Should we let them provide a string to append here (perhaps I'm just trying out a change or something and want to deploy two separate ones)